apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-thumbnail-presenter.tsx

```diff
@@ -20,13 +23,35 @@ export interface ResourceDetailThumbnailPresenterProps {
  * This is WIP. We will solidify the API as we make our first pass of the resources.
  */
 export const ResourceDetailThumbnailPresenter = ({
+    id,
+    type,
     name,
     children,
 }: ResourceDetailThumbnailPresenterProps): JSX.Element => (
     <Card>
-        <CardHeader>
-            <MultilingualTextPresenter text={name} />
-        </CardHeader>
-        <CardContent>{children}</CardContent>
+        <CardContent>
+            <Grid container spacing={1} columns={{ xs: 2, sm: 4, md: 12 }}>
+                <Grid item xs={2} sm={1} md={2}>
+                    {/* Preview will eventually include images taken from video or photos, etc. */}
+                    <ResourcePreviewIconFactory resourceType={type} size="md" />
+                </Grid>
+                <Grid item xs={2} sm={2} md={8}>
+                    {/* TODO: consider putting a standardized name property on the view models */}
+                    <Typography gutterBottom variant="h6" fontWeight="bold" color="primary">
+                        {isString(name) || isNullOrUndefined(name) ? (
+                            name
+                        ) : (
+                            <MultilingualTextPresenter text={name} />
+                        )}
+                    </Typography>
+                    {children}
+                </Grid>
+                <Grid item xs={2} sm={1} md={2} container sx={{ justifyContent: 'flex-end' }}>
+                    <ResourceNavLink
+                        internalLink={`/${routes.resources.ofType(type).detail(id)}`}
```

Comment:
For some reason, I'd prefer to pass in the link instead of the resource type. Let's talk about this when you work on the next resource's presenters.

Reply:
Can you pass  the link that includes the calls to route?

`/${routes.resources.ofType(type).detail(id)}`

Reply:
Yea. Just render that using a template literal and pass it as a string.

Reply:
Great